web/packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -22,7 +22,7 @@ import { useParams } from 'react-router';
 import useAttempt from 'shared/hooks/useAttemptNext';
 
 import { ButtonState } from 'teleport/lib/tdp';
-import { useMfa } from 'teleport/lib/useMfa';
+import { useMfaTty } from 'teleport/lib/useMfa';
```

Comment:
@Joerger why is desktop session calling `useMfaTty`? There is no TTY for desktop sessions - this is SSH-only code, is it not?